stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/MediaAttachmentViewStyle.kt

```diff
@@ -27,6 +27,7 @@ import io.getstream.chat.android.ui.message.list.adapter.view.internal.MediaAtta
 public data class MediaAttachmentViewStyle(
     public val progressIcon: Drawable,
     public val giphyIcon: Drawable,
+    public val placeHolderIcon: Drawable,
```

Comment:
typo: should be `placeholderIcon`. Could you check all `placeHolder` occurrences?